src/zksync-specifics/cheatcodes/zk-use-factory-dep.md

```diff
@@ -0,0 +1,28 @@
+## `zkUseFactoryDep`
+
+### Signature
+
+```solidity
+function zkUseFactoryDep(string calldata name) external pure;
+```
+
+### Description
+
+Enables marking a bytecode as a factory dependency for the next CREATE or CALL and cleared after.
+
+This cheatcode is useful when deploying contracts through factories, as it allows marking bytecode as a factory dependency and deploying contracts by passing their bytecode hash to the factory.
+
+### Examples
+
+```solidity
+contract Deployer {
+    // Deploy the factory
+    Factory factory = new Factory(multisigBytecodeHash);
```

Comment:
Can we have a few extra lines to explain what's happening here? Specifically: 
* `Factory` does not depend on `TwoUserMultisig` directly
* We thus need to mark it explicitly